.env

```diff
@@ -6,3 +6,13 @@ REACT_APP_BUILD_VERSION=
 # REACT_APP_TESTNET_API=https://testnet.nexus.stg.oasis.io/v1/
 REACT_APP_API=https://nexus.oasis.io/v1/
 REACT_APP_TESTNET_API=https://testnet.nexus.oasis.io/v1/
+# REACT_APP_NO_BUILD_BANNERS=true
```

Comment:
With all these entries, can we define env.d.ts?

Reply:
Declared the potential entries `global.d.ts`.